### DISCUSSION TITLE
0.8.1

### DISCUSSION THREAD

**Discussion by Barbarur** [Barbarur/NovaPoint#32] [Announcements]
### Solutions
Include Copy library files across sites.
Include bulk deletion of sites.

### Improvements
Improve UI.
Performance improvement.
BugFix.

<hr /><em>This discussion was created from the release <a href='https://github.com/Barbarur/NovaPoint/releases/tag/0.8.1'>0.8.1</a>.</em>

**Comment by TheRealStriff** [comment]
Thanks,
Seems like it is also faster deleting file versions. !
